worker/backend/backend.go

```diff
@@ -106,14 +113,75 @@ func (b *Backend) Create(gdnSpec garden.ContainerSpec) (container garden.Contain
 //
 // Errors:
 // * TODO.
-func (b *Backend) Destroy(handle string) (err error) { return }
+func (b *Backend) Destroy(handle string) error {
+	ctx := namespaces.WithNamespace(context.Background(), b.namespace)
```

Comment:
Does it make sense for this to be a context.TODO as we intend it to be replaced ?

Reply:
I am not 100% sure about this - will have to check with @cirocosta later but I _think_ this is actually the code we want

Reply:
The `context.Background` is never cancellable and is generally used as the root context, some more notes [here](https://golang.org/pkg/context/#Background)
If we have a plan to shift these cases from context.Background or context.TODO to a context with a timeout or a cancellable context, I am happy :)

Reply:
Chatted with @deniseyu @pivotal-bin-ju and we are using context with a timeout everywhere now